cmd/clusterctl/cmd/get_kubeconfig_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	_ "embed"
+	"os"
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+//go:embed testdata/existsing-kubeconfig.yaml
+var existingKubeconfig string
+
+//go:embed testdata/workload-kubeconfig.yaml
+var workloadKubeconfig string
+
+//go:embed testdata/joined-kubeconfig.yaml
+var joinedKubeconfig string
```

Comment:
```suggestion
var (
//go:embed testdata/existsing-kubeconfig.yaml
existingKubeconfig string

//go:embed testdata/workload-kubeconfig.yaml
workloadKubeconfig string

//go:embed testdata/joined-kubeconfig.yaml
joinedKubeconfig string
)
```

Just a bit easier to read.

Reply:
Ok, will fix